catalyst-gateway/bin/src/db/index/queries/rbac/get_rbac_registrations.rs

```diff
@@ -127,8 +128,9 @@ pub(crate) async fn build_reg_chain(
     Ok(Some(chain))
 }
 
-/// A helper function to return a RBAC registration from the given block and slot.
-pub(crate) async fn registration(
+/// A helper function to load a RBAC registration by the given block and slot from the
+/// `cardano-chain-follower`.
+pub(crate) async fn load_registration_from_chain(
```

Comment:
I can live with that, but I think this is against our rule about unnecessary renames. 🙃 Personally I prefer more concise naming and think that "an indexed registration" and "a (normal) registration" gives enough context, but if you prefer to have more distinct names then maybe let's call this function something like `load_cip509`. After all it is more important that these are different types and how exactly we get them.

Reply:
I would rename it, because original `registration` name does not tell anything what this function is doing.
What do you think about `load_cip509_from_chain` ? I would emphasize that this function loads data from the chain, not from the db